keras/utils/data_utils.py

```diff
@@ -186,19 +186,18 @@ def get_file(fname,
 
     if download:
         print('Downloading data from', origin)
-        progbar = None
+        enclosed = {'progbar': None}
 
-        def dl_progress(count, block_size, total_size, progbar=None):
-            if progbar is None:
-                progbar = Progbar(total_size)
+        def dl_progress(count, block_size, total_size):
+            if enclosed['progbar'] is None:
```

Comment:
just wasn't sure if "probar" -> "enclosed" goes from a local to a global variable. NVM about threads, that's out of scope. Thanks.